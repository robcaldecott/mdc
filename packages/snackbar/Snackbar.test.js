import React from "react";
import { Snackbar, SnackbarAction } from ".";
import renderer from "react-test-renderer";

it("renders", () => {
  expect(
    renderer.create(
      <Snackbar
        active
        multiline
        actionOnBottom
        alignStart
        action={<SnackbarAction>Action</SnackbarAction>}
      >
        Message
      </Snackbar>
    )
  ).toMatchSnapshot();
});
