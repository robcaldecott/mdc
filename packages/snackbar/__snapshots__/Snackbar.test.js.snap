// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  aria-atomic="true"
  aria-hidden="true"
  aria-live="assertive"
  className="mdc-snackbar mdc-snackbar--active mdc-snackbar--multiline mdc-snackbar--action-on-bottom mdc-snackbar--align-start"
>
  <div
    className="mdc-snackbar__text"
  >
    Message
  </div>
  <div
    className="mdc-snackbar__action-wrapper"
  >
    <button
      className="mdc-snackbar__action-button"
      type="button"
    >
      Action
    </button>
  </div>
</div>
`;
